Description: Disable -Wl,--hash-style=both on mips machines.
 It's not supported on mips and mipsel ports and leads to build errors.
Author: Ognyan Kulev <ogi@debian.org>
Forwarded: not-needed
Last-Update: 2013-04-24
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- bowtie-1.0.0.orig/Makefile
+++ bowtie-1.0.0/Makefile
@@ -45,7 +45,9 @@
 LINUX = 0
 ifneq (,$(findstring Linux,$(shell uname)))
     LINUX = 1
-    EXTRA_FLAGS += -Wl,--hash-style=both
+    ifeq (,$(findstring mips,$(shell uname -m)))
+        EXTRA_FLAGS += -Wl,--hash-style=both
+    endif
 endif
 
 MM_DEF = 
